modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,148 @@
+
+User-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions with secondary networks. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` and `ClusterUserDefinedNetwork` CRs. It is only available for `NetworkAttachmentDefinition` CRs that leverage secondary networks.
+====
+
+The following section highlights the supported features of the `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs when they are used as either the primary or secondary network. A separate table for the `ClusterUserDefinedNetwork` CR is also included. 
+ 
+.Primary network support matrix for `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs
+[cols="1a,1a,1a, options="header"]
+|===
+^| Network feature ^| Layer2 topology ^|Layer3 topology
+
+^| east-west traffic
+^| &#10003;
+^| &#10003;
+
+^| north-south traffic
+^| &#10003;
+^| &#10003;
+
+^| Persistent IPs
+^| &#10003;
+^| X
+
+^| Services
+^| &#10003;
+^| &#10003;
+
+^| `EgressIP` resource
+^| &#10003;
+^| &#10003;
+
+^| Multicast ^[1]^
+^| &#10003;
+^| &#10003;
+
+^| `NetworkPolicy` resource ^[2]^
+^| &#10003;
+^| &#10003;
+
+^| `MultinetworkPolicy` resource 
+^| X
+^| X
+
+|===
+1. Multicast must be enabled in the namespace, and it is only available between OVN-Kubernetes network pods. For more information about multicast, see "About multicast".
+2. When creating a `UserDefinedNetwork` CR with a primary network type, network policies must be created _after_ the `UserDefinedNetwork` CR.
+
+.Secondary network support matrix for `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs
+[cols="1a,1a,1a,1a, options="header"]
+|===
+^| Network feature ^| Layer2 topology ^|Layer3 topology ^|Localnet topology ^[1]^
+
+^| east-west traffic
+^| &#10003;
+^| &#10003;
+^| &#10003; (`NetworkAttachmentDefinition` CR only)
+
+^| north-south traffic
+^| X
+^| X
+^| &#10003;
+
+^| Persistent IPs
+^| &#10003;
+^| X
+^| &#10003; (`NetworkAttachmentDefinition` CR only)
+
+^| Services
+^| X
+^| X
+^| X
+
+^| `EgressIP` resource
+^| X
+^| X
+^| X
+
+^| Multicast
+^| X
+^| X
+^| X
+
+^| `NetworkPolicy` resource
+^| X
+^| X
+^| X
+
+^| `MultinetworkPolicy` resource
+^| &#10003;
+^| X
+^| &#10003; (`NetworkAttachmentDefinition` CR only)
+
+|===
+1. The Localnet topology is unavailable for use with the `UserDefinedNetwork` CR. It is only supported on secondary networks for `NetworkAttachmentDefinition` CRs.
+
+.ClusterUserDefinedNetwork support matrix for `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs
```

Comment:
Don't we want to say something like "Cluster wide user defined networks support ..." ? I worry that since `ClusterUserDefinedNetwork` is a CRD name, this sentence becomes a bit confusing.

Reply:
Silly copy and paste error on my end. Good catch.